system-design/app-interoperability/application-package-definition.md

```diff
@@ -17,286 +33,188 @@ The application package has the following folder structure:
 
 **Application Description Example**
 
-An example of an `margo.yaml` file is shown below:
+A simple hello-world example of an `margo.yaml` file is shown below:
 
 ```yaml
-apiVersion: margo.dev/v1 
-kind: application 
-metadata: 
-  name: my-helm-nginx-margo-app 
-  version: 0.1.0 
-  description: Example of packaging nginx as a Margo application.
-  icon: "./resources/nginx-logo-s.png" 
-  author: John Smith 
-  organization: App Devs Inc. 
-  organization-site: https://app-devs.com 
-  app-tagline: My cool example app. 
-  description-long: Long description of my cool app ... 
-  license-file: "./resources/LICENCE.txt" 
-  app-site: https://my-cool-app.com 
-spec: 
-  helm-chart: 
-    repoType: local
-    url: "./manifest/helm-chart" 
-  docker-compose: 
-    repoType: local 
-    url: "./manifest/docker-compose/compose.yaml" 
+apiVersion: margo.org/v1-alpha1
+kind: application
+metadata:
+  id: com.northstartida.hello.world
+  name: Hello World
+  description: A basic hello world application
+  version: 1.0
+  catalog:
+    application:
+      icon: ./resources/hw-logo.png
+      tagline: Northstar Industrial Application's hello world application.
+      descriptionFile: ./resources/description.md
+      releaseNotes: ./resources/release-notes.md
+      licenseFile: ./resources/license.pdf
+      site: http://www.northstar-ida.com
+      tags: ["monitoring"]
+    author:
+      - name: Roger Wilkershank
+        email: rpwilkershank@northstar-ida.com
+    organization:
+      - name: Northstar Industrial Applications
+        site: http://northstar-ida.com
+deployments:
+  - type: helm.v3
+    components:
+      - name: hello-world
+        properties:  
+          repository: oci://northstarida.azurecr.io/charts/hello-world
+          revision: 1.0.1
+          wait: true
+```
+
+An example of a `margo.yaml` file using multiple helm charts and a docker-compose file package is shown below.
+
+```yaml
+apiVersion: margo.org/v1-alpha1
+kind: application
+metadata:
+  id: com.northstartida.digitron.orchestrator
+  name: Digitron orchestrator
+  description: The Digitron orchestrator application
+  version: 1.2.1 
+  catalog:
+    application:
+      icon: ./resources/ndo-logo.png
+      tagline: Northstar Industrial Application's next-gen, AI driven, Digitron instrument orchestrator.
+      descriptionFile: ./resources/description.md
+      releaseNotes: ./resources/release-notes.md
+      licenseFile: ./resources/license.pdf
+      site: http://www.northstar-ida.com
+      tags: ["optimization", "instrumentation"]
+    author:
+      - name: Roger Wilkershank
+        email: rpwilkershank@northstar-ida.com
+    organization:
+      - name: Northstar Industrial Applications
+        site: http://northstar-ida.com
+deployments:
```

Comment:
Maybe a slightly better name is deployment-profiles. Because the way it's organized today it feels both helm and compose need to be deployed (as they are both listed under deployments) but in reality, a WOS may choose to deploy one of them.

Reply:
@arne-broering what do you think of the suggestion from @Haishi2016 here? I'm fine with using `deploymentProfiles` instead of just `deployments` if it makes the intent clearer.

Reply:
@arne-broering I liked the suggestion so I went ahead and made this change.